rpcs3/Emu/RSX/GL/GLGSRender.cpp

```diff
@@ -464,27 +442,23 @@ void nv4097_clear_surface(u32 arg, GLGSRender* renderer)
 	}
 
 	glClear(mask);
-	renderer->write_buffers();
+	write_buffers();
 }
 
-using rsx_method_impl_t = void(*)(u32, GLGSRender*);
-
-static const std::unordered_map<u32, rsx_method_impl_t> g_gl_method_tbl =
-{
-	{ NV4097_CLEAR_SURFACE, nv4097_clear_surface }
-};
-
 bool GLGSRender::do_method(u32 cmd, u32 arg)
 {
-	auto found = g_gl_method_tbl.find(cmd);
-
-	if (found == g_gl_method_tbl.end())
-	{
+	switch (cmd)
+	{
```

Comment:
Do not do it. Place back unordered_map, because much more commands would be overridden in near time